common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -850,26 +850,35 @@ public UTF8String translate(Map<Character, Character> dict) {
     return fromString(sb.toString());
   }
 
-  private int getDigit(byte b) {
-    if (b >= '0' && b <= '9') {
-      return b - '0';
+  public static class LongWrapper {
+    private long value = 0;
```

Comment:
it's for internal use only, shall we make it public? We can probably save some function calls